views/me.hbs

```diff
@@ -0,0 +1,5 @@
+<script src="/vendor/zepto/dist/zepto.min.js"></script>
+<script src="/js/main.js"></script>
+<script>
+redirectToUserPage();
```

Comment:
Call this from `main.js` using a document ready type thing.

Reply:
Should I check the URL and redirect if it matches?

Reply:
Ah yeah that would be good to 👍